fuzzers/mutators.ts

```diff
@@ -29,6 +29,26 @@ export function randomLanguage(): string {
 	}
 }
 
+// We use string values because they can be printed when reporting errors.
+export type Backend = 'expression' | 'js-codegen';
```

Comment:
Why u no enum?

Reply:
I assumed enums could only have number values. Just looked it up and I learned I can use strings too. I'm going to change it.